### ISSUE TITLE
[ISSUE] - Error when using Quest item from the quest tracker

### ISSUE THREAD

**Issue by EzeRage** [Azeroth-Pilot-Reloaded/azeroth-pilot-reloaded#356]
### Description

Addon still worked and nothing appeared to be wrong just got the error.

### World of Warcraft Flavor

Retail (Default)

### APR Status

{{"Class","Paladin"},{"Continent","Khaz Algar"},{"Time & Date","Mon Nov 11 17:29:44 2024"},{"Language","enUS"},{"Name","Norepairbil"},{"Realm","Azuremyst"},{"World Coordinates","-3369.90, 3541.60"},{"Level","80"},{"AddOn Version","v4.5.4"},{"Faction","Alliance"},{"Coordinates","58.89, 28.08"},{"Client Build","57388"},{"Client Version","11.0.5"},{"Index, Step","445, Qpart"},{"Route","2248-TWW-Isle-of-Dorn-Full"},{"Zone","Isle of Dorn"}}

### Tested with only APR

- [ ] Yes
- [X] No

### Lua Error

127x APR/APR-Core/CurrentStep.lua:710: bad argument #1 to 'SetCooldown' (Usage: self:SetCooldown(start, duration [, modRate]))
[string "=[C]"]: in function `SetCooldown'
[string "@APR/APR-Core/CurrentStep.lua"]:710: in function `UpdateStepButtonCooldowns'
[string "@APR/APR-Core/QuestHandler.lua"]:846: in function `?'
[string "@APR/APR-Core/QuestHandler.lua"]:803: in function `UpdateStep'
[string "@APR/APR-Core/QuestHandler.lua"]:1071: in function `func'
[string "@APR/APR-Core/QuestHandler.lua"]:1081: in function <APR/APR-Core/QuestHandler.lua:1060>

Locals:
(*temporary) = CurrentStepFrame_StepHolderIconButtonCooldown {
}
(*temporary) = nil
(*temporary) = 3

### Reproduction Steps

Step 443/564 on Isle of Dorn - Sojourner route.

### Screenshots

![Screenshot 2024-11-11 173159](https://github.com/user-attachments/assets/795981e9-9a31-47d0-a6d6-cbd85893ddfd)

**Comment by Meurx** [comment]
In APR/APR-Core/QuestHandler.lua on line 710 change it from

```lua
IconButton.cooldown:SetCooldown(start, duration)
```

to

```lua
IconButton.cooldown:SetCooldown(startTime, duration)
```

**Comment by Neogeekmo** [comment]
Fixed with v4.5.5
